src/App.tsx

```diff
@@ -1,24 +1,158 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos, createTodo, removeTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { Filter } from './types/Filter';
+import { ErrorAction } from './types/ErrorAction';
+import { Notification } from './components/Notification';
 
-const USER_ID = 0;
+const USER_ID = 6655;
+
+function filteredTodos(todos:Todo[], filter:Filter) {
+  const returnArr = [...todos];
+
+  switch (filter) {
+    case Filter.ACTIVE:
+      return returnArr.filter(todo => !todo.completed);
+    case Filter.COMPLETED:
+      return returnArr.filter(todo => todo.completed);
+    default:
+      return returnArr;
+  }
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<Filter>(Filter.ALL);
+  const [errorMessage, setErrorMessage]
+    = useState<ErrorAction>(ErrorAction.NONE);
+  const visibleTodos = filteredTodos(todos, filterBy);
+  const todosLeft = todos.filter(todoLeft => !todoLeft.completed).length;
+  const [tempTodo, setTempTodo] = useState<Todo>();
+  const [disabledInput, setDisabledInput] = useState(false);
```

Comment:
```suggestion
  const [isInputDisabled, setIsInputDisabled] = useState(false);
```

https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig